src/server/server_state.h

```diff
@@ -226,6 +228,8 @@ class ServerState {  // public struct - to allow initialization.
 
   absl::flat_hash_map<std::string, base::Histogram> call_latency_histos_;
   uint32_t thread_index_ = 0;
+  uint64_t used_mem_ = 0;
```

Comment:
`used_mem_cached_`  and add a comment saying it's a thread local cache of `used_mem_current`